src/pc/djui/djui_hud_utils.c

```diff
@@ -131,14 +134,29 @@ void patch_djui_hud(f32 delta) {
         djui_hud_position_translate(&translatedX, &translatedY);
         create_dl_translation_matrix(DJUI_MTX_PUSH, translatedX, translatedY, gDjuiHudUtilsZ);
 
-        // translate scale
-        f32 translatedW = scaleW;
-        f32 translatedH = scaleH;
-        djui_hud_size_translate(&translatedW);
-        djui_hud_size_translate(&translatedH);
+       // rotate
+       f32 translatedW = scaleW;
+       f32 translatedH = scaleH;
+       s32 rotation = delta_interpolate_s32(sRotation.rotation-sRotation.rotationDiff, sRotation.rotation, delta);
+       f32 pivotX = delta_interpolate_f32(sRotation.prevPivotX, sRotation.pivotX, delta);
+       f32 pivotY = delta_interpolate_f32(sRotation.prevPivotY, sRotation.pivotY, delta);
+       djui_hud_size_translate(&translatedW);
+       djui_hud_size_translate(&translatedH);
+       if (sRotation.rotationDiff != 0 || sRotation.rotation != 0) {
+           f32 pivotTranslationX = translatedW * pivotX;
+           f32 pivotTranslationY = translatedH * pivotY;
+           create_dl_translation_matrix(DJUI_MTX_NOPUSH, +pivotTranslationX, -pivotTranslationY, 0);
+           create_dl_rotation_matrix(DJUI_MTX_NOPUSH, rotation, 0, 0, 1);
+           create_dl_translation_matrix(DJUI_MTX_NOPUSH, -pivotTranslationX, +pivotTranslationY, 0);
+       }
+
+        // scale
+     // djui_hud_size_translate(&translatedW);
+     // djui_hud_size_translate(&translatedH);
```

Comment:
don't leave this please, i assume this wasn't intentional

Reply:
I agree

Reply:
while fixing this, i found that that entire section was offtab
![image](https://github.com/user-attachments/assets/f33e011a-d845-48d1-9246-51049d787e56)

thanks